internal/accounts/create.go

```diff
@@ -151,29 +149,28 @@ func create(
 func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
 	log := output.NewStdoutLogger(output.InfoLog)
 
-	log.Info(fmt.Sprintf("What name would you like to give this new account?"))
-	name := output.NamePrompt()
-
+	name := output.AccountNamePrompt() // todo check for duplicate names
 	networkName, selectedNetwork := output.CreateAccountNetworkPrompt()
+
 	// create new gateway based on chosen network
 	gw, err := gateway.NewGrpcGateway(selectedNetwork.Host)
 	if err != nil {
 		return nil, err
 	}
-	privateJsonFileName := ""
+
+	privateFile := output.Bold(fmt.Sprintf("%s.private.json", name))
+
+	log.Info(fmt.Sprintf("\n%s This command will perform the following:", output.WarningEmoji()))
+	log.Info(fmt.Sprintf("- Generate a new ECDSA P-256 public and private key pair."))
```

Comment:
Super nitpicky, but I like that the numbered steps below are indented with one space. It'd be good to make these ticks match (and same with the final summary).
![Screen Shot 2022-07-20 at 1 21 43 PM](https://user-images.githubusercontent.com/2547035/180074560-3a63a1d4-9eb2-4adf-b9fd-817cf8fb1df6.png)

Reply:
Not nitpicky at all, I agree 100% just overlooked it.